### ISSUE TITLE
Is this repo still in active development?

### ISSUE THREAD

**Issue by ajing** [mlr-org/automlr#1]

**Comment by giuseppec** [comment]
Not really. But I think it is also not "dead". Maybe you find this interesting: https://github.com/ja-thomas/autoxgboost which is very similar and under active development.

**Comment by berndbischl** [comment]
@mb706 is still working on this but more focusing on his gsoc project at the moment

**Comment by berndbischl** [comment]
why do you ask?

**Comment by ajing** [comment]
I saw auto-sklearn and tpot. So, I am wondering whether mlr has something comparable, which may worth to try.
